jetson/nav/README.md

```diff
@@ -0,0 +1,84 @@
+# <img src="../../simulators/nav/src/static/mrover.png" alt="MRover Logo" width="30"/> MRover Navigation Codebase
+
+## Table of Contents
+[Project Overview](#project-overview)<br/>
+[Top-Level Code](#Top-Level-Code)<br/>
+[Gate Search (`gate_search/`)](#Gate-Search)<br/>
+[Obstacle Avoidance (`obstacle_avoidance/`)](#obstacle-avoidance)</br>
+[Search (`search/`)](#search)<br/>
+[Variables and Utilities](#vars-and-utils)<br/>
+
+
+**Created:** Ana Warner, Oct 3, 2021
+**Updated:** A.Warner, 10/14/2021
+
+---
+
+<!----------------------------- Project Overview ----------------------------->
+## Project Overview
+The nav codebase contains logic for commanding the rover during the Autonomous Traversal task. The task consists of several legs that increase in difficulty. During a leg, the operator starts inputs the destination GPS waypoint and end behavior for the leg (search and/or gate), then starts the Auton system from the base station. The rover will first drive autonomously to the destination waypoint without responding to target data from Perception. It will avoid obstacles as they appear. Once the waypoint is reached, the rover will start a search pattern if the leg was a search leg. The search algorithm first attempts a spiral out search pattern and approaches the tag once it is found. Finally, if there is a second tag composing the gate, the gate search and traversal algorithm will start.
```

Comment:
The third sentence in this paragraph doesn't make sense. (During a leg, ...)
What does 'there is a second tag composing the gate' mean in the last sentence?